src/index.html

```diff
@@ -1,13 +1,406 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Crazybaby</title>
+    <link rel="stylesheet" href="/src/styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap" rel="stylesheet">
+    <link
+      rel="shortcut icon"
+      href="/src/images/favicon.png"
+      type="image/x-icon"
+    >
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body" id="page-top">
     <script type="text/javascript" src="scripts/main.js"></script>
+    <!-- #region header -->
+    <header class="header">
+      <div class="top-bar header__top">
+        <a class="top-bar__logo" href="#page-top">
+          <img src="./images/logo.svg" alt="Crazybaby logo">
+        </a>
+
+        <div class="top-bar__buttons-holder">
+          <div class="top-bar__button">
+            <div class="button-specs">
+              <span class="button-specs-label">SPECS</span>
+              <div class="button-specs-container">
+                <span class="button-specs-options">EN</span> / <span class="button-specs-options">UA</span>
+              </div>
+            </div>
+            <a href="#get-in-touch">
+              <div class="button-buy">BUY</div>
```

Comment:
This button is with a blue line above that. Please remove.
![image](https://github.com/mate-academy/Kickstarter/assets/37789845/2b3be368-15c0-4ae4-9569-696def8f34f8)